Tests/misc/timeTools_test.py

```diff
@@ -1,9 +1,25 @@
 from __future__ import print_function, division, absolute_import
 from fontTools.misc.py23 import *
-from fontTools.misc.timeTools import asctime
+from fontTools.misc.timeTools import asctime, timestampNow, epoch_diff
+import os
 import time
 
 
 def test_asctime():
     assert isinstance(asctime(), basestring)
     assert asctime(time.gmtime(0)) == 'Thu Jan  1 00:00:00 1970'
+
+def test_source_date_epoch():
+    os.environ["SOURCE_DATE_EPOCH"] = "150687315"
+    assert timestampNow() + epoch_diff == 150687315
+
+    # Check that malformed value fail, any better way?
```

Comment:
@khaledhosny you can use `pytest.raises()`, as I did here https://github.com/fonttools/fonttools/commit/2e4b1d05df2d720bc1f4ba03b6dacb9171116ee1

More info https://docs.pytest.org/en/latest/assert.html#assertions-about-expected-exceptions

Reply:
Thanks!